clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/security/AzureNamedAccountCredentials.groovy

```diff
@@ -81,11 +81,8 @@ public class AzureNamedAccountCredentials implements AccountCredentials<AzureCre
   }
 
   private AzureCredentials buildCredentials() {
-    AzureResourceManagerClient rmClient = new AzureResourceManagerClient(this.subscriptionId)
-    AzureNetworkClient networkClient = new AzureNetworkClient(this.subscriptionId)
-    AzureComputeClient computeClient = new AzureComputeClient(this.subscriptionId)
 
-    return new AzureCredentials(this.tenantId, this.clientId, this.appKey, rmClient, networkClient, computeClient)
+    return new AzureCredentials(this.tenantId, this.clientId, this.appKey, this.subscriptionId)
```

Comment:
we could we this as an opportunity to make this class more groovy like (remove ";" and unnecessary "return").